ci/autoscaler/set-pipeline.sh

```diff
@@ -1,12 +1,17 @@
-#!/bin/bash
+#! /usr/bin/env bash
+#
 # To run this script you need to have set up the target using
 # fly -t "autoscaler" login -c "https://bosh.ci.cloudfoundry.org" -n "app-autoscaler"
-# When running locally: ` fly -t "local" login -c "http://localhost:8080" `
+#
+# When running concourse locally: ` fly -t "local" login -c "http://localhost:8080" `
 # Then  `TARGET=local set-pipeline.sh`
 set -euo pipefail
 
-TARGET=${TARGET:-autoscaler}
+SCRIPT_RELATIVE_DIR=$(dirname "${BASH_SOURCE[0]}")
```

Comment:
This probably should be the more portable version of `"$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"` instead but this will  probably be fine for